dev/tools/gen_date_localizations.dart

```diff
@@ -88,10 +88,19 @@ Future<Null> main(List<String> rawArgs) async {
   });
   buffer.writeln('};');
 
-  buffer.writeln('const Map<String, dynamic> datePatterns = const <String, dynamic> {');
+  // Note: code that uses datePatterns expects it to contain values of type
+  // Map<String, String> not Map<String, dynamic>.
+  buffer.writeln('const Map<String, Map<String, String>> datePatterns = const <String, Map<String, String>> {');
   patternFiles.forEach((String locale, File data) {
```

Comment:
It seems the computer you ran this script on sorts files differently. Let's have `_listIntlData` sort the map by locale so the contents of `date_localizations.dart` are stable.

Reply:
Sorted `_listIntlData` result.